clang/test/CodeGen/builtins-nvptx-native-half-type-err.c

```diff
@@ -1,12 +1,605 @@
+// NOTE: Assertions have been autogenerated by utils/update_cc_test_checks.py UTC_ARGS: --version 5
```

Comment:
Don't switch tests to generated checks at the same time as changing functionality

Reply:
Well previously this test expected them all to fail, they don't fail anymore. Should I just delete the test?

Reply:
I probably should rename it as well.

Reply:
I guess this doesn't apply if it was a fail test before